test/index.spec.js

```diff
@@ -25,7 +25,7 @@ describe('babel-plugin-css-modules-transform', () => {
     }
 
     function readExpected(path) {
-        return readFileSync(resolve(__dirname, path), 'utf8');
+        return readFileSync(resolve(__dirname, path), 'utf8').trim();
```

Comment:
I ran into this too... That said, I'd say it merits a comment to explain why we'd want to trim a file's contents when used as a text fixture.